news/4.30/platform.html

```diff
@@ -75,6 +75,22 @@ <h2>Themes and Styling </h2>
     <h2>General Updates </h2>
     </td>
   </tr>
+   <tr id="eclipse_search"> <!-- https://github.com/eclipse-platform/eclipse.platform.ui/pull/1054 -->
```

Comment:
platform.html is for visible changes. [platform_isv.html](https://github.com/eclipse-platform/www.eclipse.org-eclipse/blob/master/news/4.30/platform_isv.html) is the one for such news. Sorry for not telling you earlier about that.